src/sparseml/transformers/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tools for integrating SparseML with transformers training flows
+"""
+
+# flake8: noqa
+
+try:
+    import transformers as _transformers
+
+    transformers_import_error = None
+except Exception as transformers_import_err:
+    transformers_import_error = transformers_import_err
+
+
+def _check_transformers_install():
+    if transformers_import_error is None:
+        return
+    raise ImportError(
+        "No installation of transformers found. It is recommended to use the "
+        "sparseml fork of transformers which can be installed under "
+        "sparseml[transformers] or git+https://github.com/neuralmagic/transformers.git"
+    )
+
+
+_check_transformers_install()
```

Comment:
This isn't a release blocker, but could we migrate this to the new standards where the package is importable without transformers being installed but will fail when code is executed that requires the dependencies? Makes docs compilation a lot easier and also allows for more expansion in the future for capabilities in the case of optional dependencies for different code pathways